docs/getting-started/definition.md

```diff
@@ -6,17 +6,17 @@ Definition are the basic building block of the KubeVela platform. A definition e
 
 There're four types of Definition, they're `ComponentDefinition`, `TraitDefinition`, `PolicyDefinition` and `WorkflowStepDefinition`, corresponding to the application concepts.
 
-## Sources of Definitions
+## How to get out-of-box Definitions?
 
-There're two sources of definitions:
+There're two sources to get out-of-box definitions:
 
 * Built-in definitions will be installed along with KubeVela helm chart. You can refer to the following links to learn more about built-in definitions.
     - [Component Definition](../end-user/components/references)
     - [Trait Definition](../end-user/traits/references)
     - [Policy Definition](../end-user/policies/references)
     - [Workflow Step Definition](../end-user/workflow/built-in-workflow-defs)
-* Installation of addons will install definitions if there're new capabilities contained.
-    - [Addon registry](../reference/addons/overview) contains a large catalog of addon which are maintained and verified by the KubeVela team.
+* [Addons](../reference/addons/overview), as extension of KubeVela, each addon contains a bunch of definitions along its CRD controllers.
```

Comment:
```suggestion
* [Addons](../reference/addons/overview), as extensions of KubeVela, each addon contains a bunch of definitions along its CRD controllers.
```

Reply:
Thanks! Good catch! will change in next PR